samples/highcharts/studies/dashboard-threshold/demo.css

```diff
@@ -0,0 +1,45 @@
+@import "https://code.highcharts.com/css/dashboard/gui.css";
+
+img {
```

Comment:
Do we need it as global? I'd prefer to use `.hd-component img {}` instead

Reply:
This demo is in studies and it will be discussed later.
Let's keep it as it is for now.